lib/node_modules/@stdlib/utils/jsonify-keys/docs/types/test.ts

```diff
@@ -0,0 +1,55 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import jsonifyKeys = require( './index' );
+
+
+// TESTS //
+
+// The function returns a parsed value...
+{
+	jsonifyKeys( '{beep:"boop"}' ); // $ExpectType any
+	jsonifyKeys( '"beep"' ); // $ExpectType any
+	jsonifyKeys( '22' ); // $ExpectType any
+}
+
+// The function does not compile if the first argument is a value other than a string...
+{
+	jsonifyKeys( true ); // $ExpectError
+	jsonifyKeys( false ); // $ExpectError
+	jsonifyKeys( 5 ); // $ExpectError
+	jsonifyKeys( [] ); // $ExpectError
+	jsonifyKeys( {} ); // $ExpectError
+	jsonifyKeys( ( x: number ): number => x ); // $ExpectError
+}
+
+// The function does not compile if the second argument is a value other than a function...
+{
+	jsonifyKeys( '{beep:"boop"}', true ); // $ExpectError
```

Comment:
```suggestion
	jsonifyKeys( '{ beep: "boop" }', true ); // $ExpectError
```
same for all other instances